docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -30,13 +30,29 @@ NOTE: PacketFence PKI is available on the PacketFence repository as the package
 Installing 
 ^^^^^^^^^^
 
-First of all, you will need to install the package, use the following command matching your OS.
+First of all you have the choice to install the packetfence-pki on a stand alone machine or on the PacketFence machine.
+
+For the stand alone you need to open the port 9393 in tcp to be able to reach the pki.
+
+If you install the pki on a PacketFence server, you need to do the following:
+
+ # cd /usr/local/pf
+ # vim conf/iptables.conf
+ 
+Uncomment this line and restart the service iptables:
+
+ # -A input-management-if --protocol tcp --match tcp --dport 9393 --jump ACCEPT
+ # bin/pfcmd service iptables restart
+ 
+You now need to install the package, use the following command matching your OS.
 
 Debian/Ubuntu:
+
  # sudo apt-get install packetfence-pki
 
 CentOS/RHEL:
- # yum install packetfence-pki --enablerepo=PacketFence
+
+ # yum install packetfence-pki --enablerepo="packetfence-extra, packetfence-devel"
```

Comment:
packetfence-extra is not necessarily configured.
Where do you mention how to enable it?